src/tests/babelWithOptions.spec.js

```diff
@@ -0,0 +1,37 @@
+import { transformFileSync } from '@babel/core'
+
+let cache = {}
+
+const setHash = ({ filename, id, name }) => {
+  cache[`${filename.split('/').slice(-2).join('/')} ${name}`] = id
```

Comment:
would not it be better to use `cache[id] = filename.split('/').slice(-2).join('/')` instead?
it looks like that hash -> filename is a more useful thing, so test as an example can illustrate it better

Reply:
Ye, I agree with u cause the main idea is let users find `component` by `hash`, will swap it.

BTW it's just a simple test. Mb should add examples with webpack and hash functions to docs after.